spec/setup_acceptance_node.pp

```diff
@@ -29,10 +29,10 @@
 
   # Workaround for https://github.com/theforeman/puppet-candlepin/issues/185#issuecomment-822284497
```

Comment:
I wonder if this is now solved. We don't rely on the symlink to be resolved so perhaps SELinux will understand file contexts better.